logstash-core-plugin-api/logstash-core-plugin-api.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |gem|
   gem.description   = %q{Logstash plugin API}
   gem.summary       = %q{Define the plugin API that the plugin need to follow.}
   gem.homepage      = "http://www.elastic.co/guide/en/logstash/current/index.html"
-  gem.license       = "Apache License (2.0)"
+  gem.license       = "Apache-2.0"
```

Comment:
this seems unrelated? is that new license string consistent across gemspecs?

Reply:
No it is not, I only change it when I see gem build complain.